src/main/java/org/cloudfoundry/promregator/scanner/ReactiveTargetResolver.java

```diff
@@ -49,8 +50,37 @@ public IntermediateTarget(IntermediateTarget source) {
 			this.resolvedApplicationName = source.resolvedApplicationName;
 			this.resolvedApplicationId = source.resolvedApplicationId;
 		}
-		
+
+		public IntermediateTarget(Target target) {
+			this.configTarget = target;
```

Comment:
Added new constructor to remove this work from the `resolveTarget` flow and reduce noisses